test/integration/analysis.js

```diff
@@ -7,6 +7,8 @@ var Analysis = require('../../lib/analysis');
 var BatchClient = require('../../lib/postgresql/batch-client');
 
 var testConfig = require('../test-config');
+var testHelper = require('../helper');
+var DataObservatoryMultipleMeasures = require('../../lib/node/nodes/data-observatory-multiple-measures');
```

Comment:
My original idea was not to couple the library with the reference for this very same reason: not able to create/test a node because you need the reference, but not being able to generate the reference because you don't have the node yet.

I've just created a PR over this branch with my proposal enabling testing nodes that are not part of the library, enabling testing the logic of the Factory without having to rely on Nodes from the library.

Let me know what do you think: https://github.com/CartoDB/camshaft/pull/321.